service/collector/deployment.go

```diff
@@ -15,7 +15,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/giantswarm/azure-operator/client"
-	"github.com/giantswarm/azure-operator/service/controller/v4/key"
+	"github.com/giantswarm/azure-operator/service/controller/v4patch1/key"
```

Comment:
This is rather ugly. @xh3b4sd this was added by you. what's the plan here? wait until versioning is done properly?

Reply:
I know. We have this in a couple of places. I tried to come up with something better but then thought it is not worth the effort. The operator versioning will certainly make it less weird.

Reply:
Note that we have such things in AWS as well.